src/main/app.js

```diff
@@ -59,37 +67,83 @@ fluid.defaults("gpii.app", {
             activeSet: null
         }
     },
-    // prerequisites
+    members: {
+        machineId: "@expand:{that}.installID.getMachineID()"
+    },
     components: {
-        psp: {
-            type: "gpii.app.psp",
-            createOnEvent: "onPrerequisitesReady",
+        installID: {
+            type: "gpii.installID"
+        },
+        factsManager: {
+            type: "gpii.app.factsManager",
+            priority: "after:installID"
+        },
+        surveyManager: {
+            type: "gpii.app.surveyManager",
+            priority: "after:factsManager"
```

Comment:
These two priorities should be removed as well - I don't see a hard dependency between the surveyManager and the factsManager - what would happen if the latter constructed first?